### PR TITLE
Update lifinance blockchain addresses and npm packages

### PR DESCRIPTION
**Pull request by maxklenk**
Updates the incomplete entry for LI.FI
- At LI.FI we have used multiple wallets in the past to deploy our contracts
- Ensured the currently used Factories and main [Contracts](https://docs.li.fi/smart-contracts/deployments) are included
- Added Safes that guard our contracts
- Added main [NPM packages](https://www.npmjs.com/~lifi)

### CODE REVIEW

**Review by ccerv1 — APPROVED** [review]
Fantastic! Thank you for these additions. Will need to see if the validator accepts all of them. I will help refactor anything needed. 💪 🚀

File: data/projects/l/lifinance.yaml [hunk 1]
```diff
@@ -16,70 +17,267 @@ blockchain:
     tags:
       - deployer
       - eoa
-  - address: "0x2fd525b8b2e2a69d054dccb033a0e33e0b4ab370"
+  - address: "0xCB1e9fA11Edd27098A6B2Ff23cB6b79183ccf8Ee"
```
Comment (ccerv1) [inline data/projects/l/lifinance.yaml:20]: We weren't able to identify contracts deployed from this address. If it's just a project EOA, please remove the `deployer` tag

File: data/projects/l/lifinance.yaml [hunk 2]
```diff
@@ -16,70 +17,267 @@ blockchain:
...
+    tags:
+      - deployer
+      - eoa
+  - address: "0x4577a46A3eCf44E0ed44410B7793977ffbe22CE0"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x29DaCdF7cCaDf4eE67c923b4C22255A4B2494eD7"
```
Comment (ccerv1) [inline data/projects/l/lifinance.yaml:44]: We weren't able to identify contracts deployed from this address. If it's just a project EOA, please remove the `deployer` tag. If it serves as a wallet for moving funds, you can give it a `wallet` tag (optional).

File: data/projects/l/lifinance.yaml [hunk 3]
```diff
@@ -16,70 +17,267 @@ blockchain:
...
+    tags:
+      - deployer
+      - eoa
+  - address: "0x552008c0f6870c2f77e5cC1d2eb9bdff03e30Ea0"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x4577a46A3eCf44E0ed44410B7793977ffbe22CE0"
```
Comment (ccerv1) [inline data/projects/l/lifinance.yaml:38]: We weren't able to identify contracts deployed from this address. If it's just a project EOA, please remove the deployer tag

File: data/projects/l/lifinance.yaml [hunk 4]
```diff
@@ -16,70 +17,267 @@ blockchain:
...
+      - base
+      - mainnet
+      - matic
+      - mode
+      - optimism
+      - polygon_zkevm
+      - scroll
+      - mantle
     tags:
       - contract
```
Comment (ccerv1) [inline data/projects/l/lifinance.yaml:206]: This FeeCollector only appears as a contract on arbitrum_one. Please remove the other network tags (unless I am misunderstanding how this contract works cross chain).

File: data/projects/l/lifinance.yaml [hunk 5]
```diff
@@ -16,70 +17,267 @@ blockchain:
...
-      - eoa
-      - wallet
+      - contract
+    name: FeeCollector
+
+  # Executor
+  - address: "0x2dfaDAB8266483beD9Fd9A292Ce56596a2D1378D"
+    networks:
+      - arbitrum_one
+      - base
```
Comment (ccerv1) [inline data/projects/l/lifinance.yaml:225]: We don't see this contract on Base. We've confirmed it on arbitrum_one, optimism, and mainnet. Can you please review and update the networks it's currently present on? Or you can change the tag to `any_evm` and we will include it everywhere we see it.

**Comment by ccerv1** [comment]
Hi @maxklenk 
Thanks again for your PR. I'm a big fan (and user) of LI.FI.
I just ran the validator on your PR and there's a long thread (which you can find above).
Anyway, three things to look at:

1. We have a little bug in our validator that is attributing addresses that aren't in your project file to you. You can ignore those  completely (0xdd4a12da198b367751481db286bce99717bed11f, 0xee9aa6ce83ef6aac5cd66115d2f2aa2fb011b164, 0x62c0077baa246ab15cacc4e3f9c04f7467cb4593, 0xdef1c0ded9bec7f1a1670819833240f027b25eff, 0xedcd79f34db8b78cd7a55e04dbf991ecd1a5c0f4)
2. A number of your addresses appear on various chains, including some we don't support and therefore triggered a warning. For these cases, we recommend using the `any_evm` network instead of enumerating the chains individually. We will then search on all supported chains for your address and attribute it to you whenever we find it. If we don't find it, no error.
3. There were several addresses that have specific action items I've added as comments. Some can be fixed with the `any_evm` network. Some appear to have a unique architecture that differs by chain.

Let me know if you think you can address these. If not, I can propose something for you to review tomorrow.

File: data/projects/l/lifinance.yaml (hunk 3 above)
Comment (maxklenk) [reply data/projects/l/lifinance.yaml:38]: Thanks for checking, It is one of our Dev-Wallets and I thought we may have used it to deploy some testing contracts as well, which is not the case. Will adjust the config.

File: data/projects/l/lifinance.yaml (hunk 5 above)
Comment (maxklenk) [reply data/projects/l/lifinance.yaml:225]: Thanks, I didn't know `any_evm` works like this, I expected the contract HAS to be present on all chains. It makes it easier for us to maintain it if we can just set it to `any_evm`.

File: data/projects/l/lifinance.yaml (hunk 4 above)
Comment (maxklenk) [reply data/projects/l/lifinance.yaml:187]: Updated.

**Comment by maxklenk** [comment]
@ccerv1 thanks for running the validator and your feedback. I adjusted my branch accordingly. Please rerun the script and let me know if it looks fine now.

**Review by ccerv1 — APPROVED** [review]
TYSM!

**Comment by ccerv1** [comment]
Hi @maxklenk 
I went ahead and did a clean rebase and implemented some of the remaining changes that were in your PR. There were 4 tags that were failing.

You can view my merged PR [here](https://github.com/opensource-observer/oss-directory/assets/42869436/387bf604-e5a9-414b-b71d-3a2afc7f7391). 

Feel free to rebase against my new version and check. If all is good, we can close this out. Thank you very much.

Grateful that LI.FI exists!

**Comment by maxklenk** [comment]
@ccerv1 Awesome, thank you. I will close that PR, your changes look fine.
The link you shared shows somewhere else, please check if that is what you wanted to link to.

**Pull request closed without merging**